src/components/button-swap/button-swap.twig

```diff
@@ -0,0 +1,28 @@
+{#
+  This pattern currently only supports a start icon.
+  End icon support can be added in the future if needed.
+#}
+<section role="region" data-testid="test-id" class="c-button-swap js-c-button-swap{% if class %} {{ class }}{% endif %}">
```

Comment:
Is this role necessary? My impression was `section` implicitly got the region role?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Region_role

Reply:
Only if it has an accessible name: https://w3c.github.io/html-aam/index.html#el-section

Reply:
Ahhh good call. So this gives it a role without a name? Should we give it a name instead? I wonder how much benefit an unnamed region adds here?

Reply:
I didn't know an accessible name was required for the `section` element to have a role of `region`! 🎓